apps/bandwagon/forms.py

```diff
@@ -143,6 +145,12 @@ def __init__(self, *args, **kw):
                 self.instance.type in amo.COLLECTION_SPECIAL_SLUGS):
             del self.fields['slug']
 
+    def clean(self):
+        if self.cleaned_data['your_name']:
+            raise forms.ValidationError(
```

Comment:
This might not want to raise a Validation error. It should probably look like a success and be quietly binned.